An Untitled Mining Game/An Untitled Mining Game/objects/obj_crafting_button/Step_0.gml

```diff
@@ -0,0 +1,22 @@
+var _hover = scr_get_hover(width, height);
+var click = _hover and mouse_check_button_pressed(mb_left);
+var struct_in_use = struct_id.in_use;
+
+
+hover = lerp(hover, _hover, 0.1);
+
+if (_hover and click and (!struct_in_use)) {
+	if(scr_have_mats(input)) {
+		//Mark in use
+		struct_id.in_use = true;
+		
+		//Set speed
+		struct_id.base_speed = crafting_time;
+		var struct_level = struct_id.structure_level;
+		struct_id.wait_time = max(crafting_time - (6 * struct_level), 6);
```

Comment:
Why is the number 6 chosen here?

Reply:
It's a 60 fps game, so 6 represents 1/10th of a second.